src/commands/toolbox/token/create.ts

```
@@ -21,109 +22,206 @@
+                    case 'image':
+                        return '--image: Path to the token image file (PNG, JPG, or GIF)';
+                    case 'mint':
+                        return '--mint-amount: Initial amount of tokens to mint (must be greater than 0)';
+                    default:
+                        return `--${flag}: Required field`;
+                }
+            });
+            
+            throw new Error(
+                `Missing required information:\n${flagDescriptions.join('\n')}\n\n` +
+                'Please provide all required information or use --wizard for interactive mode.'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)